cypress/e2e/template/HSMTemplateV2.spec.ts

```
@@ -0,0 +1,339 @@
+describe('HSM Template V2', () => {
+  const hsmTemplateName = 'sample_hsm_v2_' + +new Date();
```

ESLint

[error] Use Date.now() to avoid allocating a new Date object.

(e18e/prefer-date-now)